.github/workflows/packages-upload-images.yml

```diff
@@ -0,0 +1,224 @@
+
+      - name: Checkout wazuh/wazuh repository
+        uses: actions/checkout@v4
+
+      - name: Get changed files
+        uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            pkg_deb_manager_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/amd64/manager/**'
+              - 'packages/debs/utils/**'
+            pkg_rpm_manager_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/amd64/manager/**'
+              - 'packages/rpms/utils/**'
+            pkg_deb_agent_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/amd64/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_deb_agent_builder_i386:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/i386/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_rpm_agent_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/amd64/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_agent_builder_i386:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/i386/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_legacy_builder_amd64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/amd64/legacy/**'
+              - 'packages/rpms/utils/**'      
+            pkg_rpm_legacy_builder_i386:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/i386/legacy/**'
+              - 'packages/rpms/utils/**'
+            pkg_deb_agent_builder_arm64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/arm64/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_deb_agent_builder_armhf:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/armhf/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_rpm_agent_builder_arm64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/arm64/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_agent_builder_armhf:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/armhf/agent/**'
+              - 'packages/rpms/utils/**'       
+            compile_windows_agent:
+              - 'packages/windows/**'
+            commom_wpk_builder:
+              - packages/wpk/wpkpack.py
```

Comment:
Why don't the next paths have single quotes? Maybe we should add them just to be consistent with the previous ones.